crates/viewer/re_component_ui/src/recording_uri.rs

```diff
@@ -35,7 +38,10 @@ pub fn singleline_view_recording_uri(
             match data_source.stream(None) {
                 Ok(re_data_source::StreamSource::LogMessages(rx)) => _ctx
                     .command_sender()
-                    .send_system(SystemCommand::AddReceiver(rx)),
+                    .send_system(SystemCommand::AddReceiver {
```

Comment:
Wait, do we generate both a `egui::OutputCommand::OpenUrl`, AND explicitly handle it here?

So that if somone installs an OS handler for `rerun://` URLs, this will open the urls twice?

An alternative is to catch the `rerun://` URLs here instead:

https://github.com/rerun-io/rerun/blob/8242c88d5511e840710f86bace673158eb83a290/crates/viewer/re_viewer/src/app_state.rs#L773-L787

That way, `rerun://` URLs will work _anywhere in our UI_. Users could even put `rerun://` in their arrow data, and we will show them as URLs, thanks to
https://github.com/rerun-io/rerun/blob/8242c88d5511e840710f86bace673158eb83a290/crates/viewer/re_ui/src/ui_layout.rs#L102-L108

Reply:
> Wait, do we generate both a `egui::OutputCommand::OpenUrl`, AND explicitly handle it here?

It seems so indeed, good catch. We could get rid of the `RecordingUri` component altogether!

Reply:
Oh, actually no, this did not double-open the URL, because it's using `Ui::link()`, which doesn't trigger a `OpenUrl` event.